front/lib/api/assistant/configuration.ts

```diff
@@ -535,10 +588,76 @@ export async function getAgentConfigurations<V extends "light" | "full">({
     });
   }
 
-  return globalAgentsPromise.then((globalAgents) => [
-    ...agentConfigurationTypes,
-    ...globalAgents,
+  return agentConfigurationTypes;
+}
+
+export async function getAgentConfigurations<V extends "light" | "full">({
+  auth,
+  agentsGetView,
+  agentPrefix,
+  variant,
+  limit,
+  sort,
+}: {
+  auth: Authenticator;
+  agentsGetView: AgentsGetViewType;
+  agentPrefix?: string;
+  variant: V;
+  limit?: number;
+  sort?: SortStrategy;
+}): Promise<
+  V extends "light" ? LightAgentConfigurationType[] : AgentConfigurationType[]
```

Comment:
The return type is broken here, as we initialize `agentConfigurationType` as `AgentConfigurationType`. Also reading the code it looks like `getGlobalAgents` always return an `AgentConfigurationType` and does not provide any `LightAgentConfigurationType`.